be/src/olap/tablet_schema.cpp

```diff
@@ -465,6 +466,11 @@ void TabletSchema::append_column(TabletColumn column, bool is_dropped_column) {
     if (column.is_nullable()) {
         _num_null_columns++;
     }
+    if (UNLIKELY(column.name() == DELETE_SIGN)) {
+        _delete_sign_idx = _num_columns;
+    } else if (UNLIKELY(column.name() == SEQUENCE_COL)) {
+        _sequence_col_idx = _num_columns;
+    }
```

Comment:
I am not sure delte sign or seq col can be enabled by alter statement.  If  a table is created without 2 columns and then they are enabled by a alter statement. delete sign index and schema does not match for read?

Reply:
I can't find the alter statement in doris doc. But it doesn't match after light schema change, because the hidden columns are always in the end. So when read and load, BE need to set their idx.